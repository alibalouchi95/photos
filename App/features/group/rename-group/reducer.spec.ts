import {
  renameGroupReducer as reducer,
  renameGroupActions as actions
} from './'

const threadId = 'id'
const name = 'name'
const initialState = reducer(undefined, {} as any)

describe('renaming groups', () => {
  describe('initial state', () => {
    it('should match snapshot', () => {
      expect(initialState).toMatchSnapshot()
    })
  })
  describe('request to rename group', () => {
    it('should manage async renaming the group', () => {
      const state0 = reducer(initialState, actions.renameGroup.request({ threadId, name }))
      expect(state0[threadId]).toBeDefined()
      const state1 = reducer(state0, actions.renameGroup.success)
      expect(state0[threadId]).toBeUndefined()
    })
  })
})
