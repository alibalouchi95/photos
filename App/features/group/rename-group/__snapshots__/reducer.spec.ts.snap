// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renaming groups initial state should match snapshot 1`] = `Object {}`;
